common/web/keyboard-processor/src/keyboards/defaultLayouts.ts

```diff
@@ -535,15 +535,22 @@ export class Layouts {
     nextlayer: 'caps'
   }
 
-  static dfltShiftRotaDefault: LayoutSubKey = {
+  static dfltShiftToDefault: LayoutSubKey = {
     // Needs to be something special and unique.  Typing restricts us from
     // using a reserved key-id prefix, though.
-    id: "T_*_MT_SHIFT_ROTA_TO_DEFAULT",
+    id: "T_*_MT_SHIFT_TO_DEFAULT",
     text: '*Shift*',
     sp: 1,
     nextlayer: 'default'
   }
 
+  static dfltShiftToShift: LayoutSubKey = {
+    id: "T_*_MT_SHIFT_TO_SHIFT",
+    text: '*Shift*',
```

Comment:
Do we use `*Shifted*` on the shift layer?

Reply:
There doesn't technically _need_ to be text for the keycap here, since modifier keys don't get multitap preview hints.  The main reason I have anything here is that a string is required as part of the standard subkey-structure typing.